src/seedu/addressbook/data/AddressBook.java

```diff
@@ -54,6 +54,13 @@ public void removePerson(ReadOnlyPerson toRemove) throws PersonNotFoundException
         allPersons.remove(toRemove);
     }
 
+    /**
+     * Sort persons in address book in alphabetical order of their name.
+     */
+    public void sortPersons(){
```

Comment:
It shouldnt it be sortPersonsbyName ? Since i see that you are sorting the persons in addressbook by name.